Model/src/BoardPackage/BoardSquare.java

```diff
@@ -1,7 +1,93 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
+import GamePackage.Player;
+
+import java.util.ArrayList;
+
+ /********************************************************************
+  * The BoardSquare Interface is the base class for all board squares
+  * which will make up the board. It tracks variables universal to
+  * all board squares and has methods for getting and setting those
+  * variables.
+  *
+  * @author David Baas
+  * @version 1.0
+ *********************************************************************/
 public class BoardSquare {
+    /**A String to track the name of the BoardSquare*/
+    protected String name;
+
+    /**An ArrayList of Players that tracks what players are on this
+     * square.
+     */
+    protected ArrayList<Player> occupiedBy;
+
+    /*****************************************************************
+     * Constructor instantiates the variables and sets the name of the
+     * BoardSquare to what is input as a parameter.
+     *
+     * @param name String that will be the name of the BoardSquare.
+     ****************************************************************/
+    public BoardSquare(String name) {
+        this.name = name;
+        occupiedBy = new ArrayList<Player>(6);
```

Comment:
The 6 could be a final variable since we want the max amount of players to be final.